docker-build-package

```diff
@@ -143,9 +143,11 @@ echo
 docker run --rm \
         --volume $(pwd)/recipes:/recipes \
         --volume $(pwd)/${output_dir}:/output \
+        --volume $(pwd)/mender-deb-package:/script \
         -e MENDER_PRIVATE_REPO_ACCESS_TOKEN="${MENDER_PRIVATE_REPO_ACCESS_TOKEN}" \
         -e MENDER_PRIVATE_REPO_ACCESS_USER="${MENDER_PRIVATE_REPO_ACCESS_USER}" \
         ${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH} \
+        /script \
```

Comment:
I would change that name to something else, as much as I love scripts :) I also love other cool things.

Reply:
Like what? Please suggest something specific.

Reply:
> Like what? Please suggest something specific.

you know naming. let's do it like this: if you are ok with this name, lets keep it.